specification/cognitiveservices/HealthInsights/healthinsights.trialmatcher/route.trialmatcher.tsp

```diff
@@ -21,7 +21,7 @@ interface TrialMatcher {
     "./examples/SuccessfulTrialMatcherResponse.json",
     "SuccessfulTrialMatcherGetAnalyzeStatus"
   )
-  getJob is Azure.Core.ResourceRead<TrialMatcherResult>;
+  getJob is StandardOperations.ResourceRead<TrialMatcherResult>;
```

Comment:
I don't think we should have this change here.  This is a breaking change we removed